src/common/outputService/deploySummaryBuilder.ts

```diff
@@ -269,3 +273,73 @@ class SoupDeploySummaryOutputService<T extends OutputFlags> extends VersionedOrS
     this.workItems = workItems;
   }
 }
+
+/**
+ * Service class used to print the operation summary of an validate-deploy
+ *
+ */
+class ValidateDeploySummaryOutputService<T extends OutputFlags> extends DeploySummaryOutputService<T> {
```

Comment:
Just a thought...maybe add "Versioned" into this name....someday we might support AdHoc, which will be different